Plate_2_data/4.processing_features/processing_features_plate2.sh

```diff
@@ -9,4 +9,8 @@ conda activate interstellar_data
 jupyter nbconvert --to python --output-dir=scripts/ *.ipynb
 
 # run the python scripts in order (from convert+merge, annotate, normalize, and feature select)
-python scripts/merge_sc_plate2.py && python scripts/annotate_sc_plate2.py && python scripts/normalize_sc_plate2.py && python scripts/feature_select_sc_plate2.py
+python scripts/0.merge_sc_plate2.py 
+python scripts/1.annotate_sc_plate2.py
+python scripts/2.combine_sc_runs_plate2.py 
+python scripts/3.normalize_sc_plate2.py 
+python scripts/4.feature_select_sc_plate2.py
```

Comment:
This will end up being replaced with CytoSnake once it is ready! It would be doing this except for combining the runs, which we had to add due to the power interruption.

I do agree that in the future to think about this but maybe for CellProfiler? 🤔